packages/mock/src/msw/index.ts

```diff
@@ -53,13 +53,19 @@ export const generateMSWImports: GenerateMockImports = ({
   isAllowSyntheticDefaultImports,
   options,
 }) => {
-  return generateDependencyImports(
+  const importBlock = generateDependencyImports(
     implementation,
     [...getMSWDependencies(options), ...imports],
     specsName,
     hasSchemaDir,
     isAllowSyntheticDefaultImports,
   );
+
+  const seed = options?.seed;
```

Comment:
I think faker's concerns should be grouped in faker-related files like `faker/getters`.